EntityFramework.Utilities/EntityFramework.Utilities.SqlServer/SqlServerBatchOperation.cs

```diff
@@ -29,8 +29,9 @@ public interface ISqlServerBatchOperationBase<TContext, T> where T : class
 
     public interface ISqlServerBatchOperationFiltered<TContext, T>
     {
-        Task<int> DeleteAsync(SqlServerDeleteSettings settings = null);
-        Task<int> UpdateAsync<TP>(Expression<Func<T, TP>> prop, Expression<Func<T, TP>> modifier, SqlServerUpdateSettings settings = null);
+        // // TODO: Full EF Core 3 support
```

Comment:
FYI: I left this comment on every place where I disabled support for batch delete and update.